src/main/java/org/dita/dost/invoker/Main.java

```diff
@@ -767,6 +788,19 @@ private void handleArgUninstall(final Deque<String> args) {
         }
         uninstallId = value;
     }
+    
+    /** Handle the --plugins argument */
+    private void printPlugins() {
+        final List<String> installedPlugins = getInstalledPlugins();
+        if (installedPlugins != null) {
+            System.out.println("The following DITA-OT plugins are installed:");
```

Comment:
I'm not sure if I want to have this line output and each ID prefixed with spaces. My preference would be to just have one ID per line, without any prefixes. That way the output could be piped to other processes without parsing out the pretty printing.

Reply:
I don't have a strong feeling on it, either is fine. 

When I got the first one working it didn't have spaces, and that seemed to look a little odd in my DOS shell. I put in the 3 spaces because they matched the `--help` output. But ... yeah it's easy to convince me to remove them here.

Reply:
The question is, for what is the output used for. In the original request

> This would also allow for scripts that can check dependencies and current installation status in support of coordinated installation of multiple plugins.

Reply:
@robander should we then change the output be plumbing output, i.e. only output the plug-in IDs or trantype names, without prefix?

Reply:
I think that's probably a good idea -- just generate the list, one per line, no extra spacing.